.github/workflows/sync-to-cms.yml

```diff
@@ -0,0 +1,27 @@
+name: Sync Plugins to CMS
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  sync-to-cms:
+    # Only run if the merged PR wasn't an automated PR for synching from the cms to the repo
+    if: github.event.pull_request.merged && !contains(github.event.pull_request.labels.*.name, 'cms_sync')
```

Comment:
@estephinson @tzmanics, this is the guard to prevent a potential infinite loop of sync repo to cms/cms to repo I mentioned. Ed or whoever will need to reapprove the PR.